### ISSUE TITLE
`max-len` is not allowing both `ignoreStrings` and `ignoreTemplateLiterals` 

### ISSUE THREAD

**Issue by sundeepgupta** [eslint/eslint#7264]
* **ESLint Version:** 3.6.1 with eslint-config-google 0.6.0
* **Node Version:** 6.3.1
* **npm Version:** 3.10.5

**What parser (default, Babel-ESLint, etc.) are you using?** default

**Please show your full configuration:**
```json
{
    "extends": "google",
    "installedESLint": true,
    "globals": {"rootRequire": true},
    "rules": {
      "no-trailing-spaces": 0,
      "eol-last": 0,
      "require-jsdoc": 0,
      "no-warning-comments": [2, {"terms": ["fixme"]}],
      "no-multiple-empty-lines": [1, {"max": 3}],
      "camelcase": 0,
      "quote-props": ["error", "as-needed"],
      "max-len": ["error", {"ignoreTemplateLiterals": true}, {"ignoreStrings": true}]
    }
}

```

**What did you do? Please include the actual source code causing the issue.**
With the above `max-len` config, these two lines produce the `max-len` error. I expected no errors.

```js
const longString = 'bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla';
const longTemplateLiteral = `bla bla bla bla bla bla bla bla bla bla bla bla bla bla`;
console.log(longString, longTemplateLiteral); // To prevent he `no-unused-vars` error
```

If I delete the `ignoreStrings` option, then there is no error on `longTemplateLiteral` line. If I instead delete the `ignoreTemplateLiterals` option, then there is an error on the `longString` line. It seems I can only include one of these options, not both.

**Comment by platinumazure** [comment]
I think your configuration is wrong. You need to combine the options object in your configuration.

```json
{
    "rules": {
      "max-len": ["error", {"ignoreTemplateLiterals": true, "ignoreStrings": true}]
    }
```

**Comment by alberto** [comment]
@platinumazure is right, so closing

**Comment by sundeepgupta** [comment]
:facepalm: sorry for the false alarm
